src/docs/automation-and-resiliency/request-quota-adjustment-for-openshift-project-set.md

```diff
@@ -216,14 +216,14 @@ This will increase the chances of us approving your request quickly (or helping
 > When my prod app is under high load, the HPA tries to start more pods than the quota allows, and then it gets stuck trying to spin up a pod that isn't allowed to start because the namespace has used up all of its quota.
 > 
 > **Outline the current and desired states of the relevant OpenShift objects.**
-> My frontend app pods use 250m CPU limit each. Currently, the HPA can spin up 6 of these before I hit the CPU limit quota for my namespace. I don't know how many pods the HPA would attempt to spin up if given more quota, so I'm not sure how much CPU limit I should expect to use. For now, I am requesting a single step increase in my CPU quota, and will revisit if I continue to encounter this problem.
+> My frontend app pods use 100m CPU request each. Currently, the HPA can spin up 6 of these before I hit the CPU request quota for my namespace. I don't know how many pods the HPA would attempt to spin up if given more quota, so I'm not sure how much CPU request I should expect to use. For now, I am requesting a single step increase in my CPU quota, and will revisit if I continue to encounter this problem.
```

Comment:
```suggestion
> My frontend app pods use 100m CPU request each. Currently, the HPA can spin up 6 of these before I hit the CPU request quota for my namespace. I don't know how many pods the HPA would attempt to spin up if given more quota, so I'm not sure how much CPU request I should expect to use. For now, I am requesting a single core increase in my CPU quota, and will revisit if I continue to encounter this problem.
```

We should remove language about increase "steps" because, with the new freeform input, we won't be using quota steps any longer.